core/src/epicli/cli/engine/ansible/AnsibleInventoryUpgrade.py

```diff
@@ -104,7 +104,7 @@ def upgrade(self):
             self.cluster_model = select_single(self.manifest_docs, lambda x: x.kind == 'epiphany-cluster')
 
         # Merge manifest cluster config with newer defaults
-        default_cluster_model = load_yaml_obj(data_types.DEFAULT, 'common', 'epiphany-cluster')
+        default_cluster_model = load_yaml_obj(data_types.DEFAULT, 'any', 'epiphany-cluster')
```

Comment:
epiphany-cluster.yml is located in different folders. Why `any` is used in this place?